nucliadb_telemetry/nucliadb_telemetry/logs.py

```diff
@@ -0,0 +1,192 @@
+# Copyright (C) 2021 Bosutech XXI S.L.
+#
+# nucliadb is offered under the AGPL v3.0 and as commercial software.
+# For commercial licensing, contact us at info@nuclia.com.
+#
+# AGPL:
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# inspired from datadog integrations like https://pypi.org/project/JSON-log-formatter/
+import logging
+from copy import copy
+from datetime import datetime
+from typing import Any
+
+import orjson
+import pydantic
+from opentelemetry import trace
+from opentelemetry.trace.span import INVALID_SPAN
+
+from nucliadb_telemetry.settings import LogSettings
+
+_BUILTIN_ATTRS = (
+    # list of all possible args
+    "args",
+    "asctime",
+    "created",
+    "exc_info",
+    "exc_text",
+    "filename",
+    "funcName",
+    "levelname",
+    "levelno",
+    "lineno",
+    "module",
+    "msecs",
+    "message",
+    "msg",
+    "name",
+    "pathname",
+    "process",
+    "processName",
+    "relativeCreated",
+    "stack_info",
+    "thread",
+    "threadName",
+)
+
+_OUTPUT_PROPERTIES = {
+    "filename": "filename",
+    "funcName": "funcName",
```

Comment:
why only this one is in camelcase?

Reply:
Because these are the standard property names. The only one I changed was `levelname` -> `severity` for gcp integration. Let me change so it's less confusing.